src/components/base/Dialogs/AlertDialog.tsx

```diff
@@ -127,6 +162,29 @@ const SwapConfirmation = (props: SwapConfirmationProps) => {
           </AlertDialogFooter>
         </AlertDialogContent>
       </AlertDialog>
+      {confirmationStep === SwapConfirmationStep.SIGN ? (
```

Comment:
We should pass the dialog messaging in as a prop to the AlertDialog component. We'll be using the dialog components for flows other than the Swap confirmation.